### ISSUE TITLE
Support for subdivision flags

### ISSUE THREAD

**Issue by Kenyon-J** [twitter/twemoji#407]
Are there any plans to provide support for flags such as;
 SI-090 Flag for St Piran,
1F3F4 - E0073 - E0069 - E0030 - E0039 - E0030 - E007F

There are a handful of these flags that I believe would enhance the variety of websites that twemoji can be applied to and at the very least would satiate my patriotic thirst

**Comment by Keijtta** [comment]
I agree that subdivisions should be added. I've created a pull request #411 where the subdivions for United States is added. I started with this country as it seems to be the most popular choise. After it has been approved, we can start on the next country.
